bitnami/jaeger/templates/_helpers.tpl

```diff
@@ -94,6 +94,11 @@ Create a container for checking cassandra availability
   {{- if $block.containerSecurityContext.enabled }}
   securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" $block.containerSecurityContext "context" .context) | nindent 4 }}
   {{- end }}
+  {{- if .context.Values.waitForDBInitContainer.resources }}
+  resources: {{- toYaml .context.Values.waitForDBInitContainer.resources | nindent 4 }}
+  {{- else if ne .context.Values.waitForDBInitContainer.resourcesPreset "none" }}
+  resources: {{- include "common.resources.preset" (dict "type" .context.Values.waitForDBInitContainer.resourcesPreset) | nindent 4 }}
+  {{- end }}
```

Comment:
As the cqlsh image is only used here and in the migrate-job.yaml file, what about if you include the resources and resourcesPreset parameters under cqlshImage.resources and cqlshImage.resourcesPreset? This way, the values.yaml file won't include an entire section just for these 2 parameters. 

```suggestion
  {{- if .context.Values.cqlshImage.resources }}
  resources: {{- toYaml .context.Values.cqlshImage.resources | nindent 4 }}
  {{- else if ne .context.Values.cqlshImage.resourcesPreset "none" }}
  resources: {{- include "common.resources.preset" (dict "type" .context.Values.cqlshImage.resourcesPreset) | nindent 4 }}
  {{- end }}
```